buildbot/master/files/config/environments.py

```diff
@@ -85,8 +99,9 @@ def __add__(self, other):
     'PKG_CONFIG_ALLOW_CROSS': '1',
 })
 
-
-build_arm32 = build_arm + Environment({
+# Unset SERVO_CACHE_DIR to test our download code for host and cross targets.
+# Use arm32 because it is the fastest cross builder.
+build_arm32 = build_arm.without(['SERVO_CACHE_DIR']) + Environment({
```

Comment:
Right, but IIRC the default value for this is `.servo` in the current directory, which on Buildbot should be the build directory, which Buildbot wipes during the git step at the start of every build. So by not setting the env var we should get option 1.